file-formats/chdo/type/zif_aff_chdo_v1.intf.abap

```diff
@@ -0,0 +1,94 @@
+INTERFACE zif_aff_chdo_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! <p class="shorttext">Table or Structure</p>
+    "! Table or structure
+    BEGIN OF ty_tablestruc,
+      "! <p class="shorttext">Name</p>
+      "! Name of table or structure
+      "! $required
+      name                   TYPE c LENGTH 30,
+      "! <p class="shorttext">Log Multiple Changes</p>
+      "! Specifies whether multiple changes can be logged
+      "! $showAlways
+      multiple_changes        TYPE abap_bool,
```

Comment:
I'm not sure if this has been discussed, yet:
All the other components of type boolean below have 'log' in their names. According to the title and description, also this one has a relation to logging. Should the name be sth. like 'log_multiple_changes'?

Reply:
No, hasn't been discussed. I'll ask the team

Reply:
I have reached out to the object type owners, yesterday. Haven't got an answer, yet. 

@wurzka What do you think? Shall we merge this PR and rename the field later, if they want to change it?

Reply:
Yes, no need to keep it open until they answer